Lib/test/test_typing.py

```diff
@@ -838,6 +838,143 @@ class C(Generic[Unpack[Ts]]): pass
         self.assertNotEqual(C[Unpack[Ts1]], C[Unpack[Ts2]])
 
 
+class TypeVarTuplePicklingTests(BaseTestCase):
+    # These are slightly awkward tests to run, because TypeVarTuples (and things
+    # made generic using TypeVarTuples) are only picklable if defined in the
+    # global scope. We therefore need to push various things defined in these
+    # tests into the global scope with `global` statements at the start of each
+    # test.
+
+    def test_typevartuple_can_be_pickled_without_error(self):
```

Comment:
This test seems redundant with the next one

Reply:
Fair, I was feeling a bit uncertain about this myself - recently I've been erring on the side of trying to separate out tests for different behaviour as much as possible, but happy to get rid of these if you're think they're more trouble than they're worth :)